<?xml version="1.0"?>
<launch>

    <param name="maxlen"          type="double" value="100.0" />    
    <param name="minlen"          type="double" value="0.0" />    
    <param name="pcTopic"         type="str"    value="/velodyne_points" />
    
    <param name="x_resolution"    type="double" value="0.25" />
    <param name="y_interpolation" type="double" value="4.0" />
    <param name="ang_Y_resolution"    type="double" value="2.05" />
 
    <param name="ang_ground"    type="double" value="0.0" />
    <param name="max_var"    type="double" value="5.0" />
    <param name="filter_output_pc"    type="bool" value="true" />

    <group ns="matrix_file">
        <rosparam file="$(find lidar_camera_fusion)/cfg/cfg_params.yaml" />
    </group>    
    <node pkg="lidar_camera_fusion" type="interpolated_node" name="interpolated_node" output="screen"/>

<!-- rviz-->
<node type="rviz" name="rviz" pkg="rviz" args="-d $(find lidar_camera_fusion)/rviz/interpoled.rviz" />
    
  </launch>
